PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -6,10 +6,18 @@ import SharedPocketKit
 import SwiftUI
 
 class AccountViewModel: ObservableObject {
+    static let ToggleAppBadgeKey = "AccountViewModel.ToggleAppBadge"
     private let appSession: AppSession
+    private let userDefaults: UserDefaults
+    private let notificationCenter: NotificationCenter
 
-    init(appSession: AppSession) {
+    @AppStorage("settingsRowToggleAppBadge")
```

Comment:
I'm not sure we have to include "settingsRowToggle" in the key here, as that's just an implementation detail. We should decide on settings key naming, so that we can form a basic pattern. 

As an example, the reader settings currently use `readerFontSizeAdjustment` and `readerFontFamily`, so I'm thinking maybe something along the lines of `settingsAppBadge` or `Settings.appBadge`.

Pinging @cyndichin for additional thoughts.

Reply:
I think we should keep it consistent with our `UserDefaults` convention, so I think `Settings.appBadge`. I also don't mind `Settings.appBadgeToggle` since it gives insight that it's a true or false value too.

Reply:
ill udpate to `Settings.appBadgeToggle`